.moon/toolchain.yml

```diff
@@ -14,10 +14,6 @@ node:
   # Accepts "npm" (default), "pnpm", or "yarn".
   packageManager: 'yarn'
 
-  # The version of the package manager (above) to use.
```

Comment:
you can probably bring this back now with the moon upgrade

Reply:
great call, looks like it works now!